core/src/main/scala/io/estatico/confide/StandardInstances.scala

```diff
@@ -49,4 +49,15 @@ trait StandardInstances {
   implicit def list[A : FromConf]: FromConf[List[A]] = iter[A].map(_.toList)
 
   implicit def vector[A : FromConf]: FromConf[Vector[A]] = iter[A].map(_.toVector)
+
+  implicit def fromConfOption[A](implicit ev: FromConf[A]): FromConf[Option[A]] = {
+    new FromConf[Option[A]] {
+      override def get(config: Config, path: String): Option[A] = {
+        if (config.hasPath(path))
+          Some(ev.get(config, path))
+        else
+          None
+      }
+    }
+  }
```

Comment:
Minor nitpick, you could probably condense the `if` to one line. The way it currently is indented someone might accidentally add a statement somewhere in there and it'd not execute properly.